programs/voter-stake-registry/tests/test_basic.rs

```diff
@@ -163,5 +165,31 @@ async fn test_basic() -> Result<(), TransportError> {
     let balance_after_withdraw = voter.deposit_amount(&context.solana, 0).await;
     assert_eq!(balance_after_withdraw, 0);
 
+    let lamports_before = context
+        .solana
+        .context
+        .borrow_mut()
+        .banks_client
+        .get_balance(voter_authority.pubkey())
+        .await?;
+    context
+        .addin
+        .close_voter(&registrar, &voter, &mngo_voting_mint, &voter_authority)
+        .await?;
+    let lamports_after = context
+        .solana
+        .context
+        .borrow_mut()
+        .banks_client
+        .get_balance(voter_authority.pubkey())
+        .await?;
+    let token_rent = context.solana.rent.minimum_balance(TokenAccount::LEN);
+    let voter_rent = context
+        .solana
+        .rent
+        .minimum_balance(std::mem::size_of::<Voter>());
+    let tolerance = 60_000;
```

Comment:
exact diff was 55680, I think sol tx fee is 0.000005 no? I would like to make this more deterministic instead of just slapping a tolerance, somehow we could compute the toelrance somehow?

Reply:
I don't know where this comes from either.